clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -84,8 +85,9 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                   Instant.ofEpochSecond(
                       auditLogRequest.getTimestamp().getSeconds(),
                       auditLogRequest.getTimestamp().getNanos()))
-              .setResource(MonitoredResource.newBuilder(MONITORED_RESOURCE_TYPE).build())
+              // .setResource(MonitoredResource.newBuilder(MONITORED_RESOURCE_TYPE).build())
```

Comment:
Is this intended?

Reply:
I have not finished the change. I'm on the debugging process. No need to take a look at it now. But this change is intended. I found that our go code https://github.com/abcxyz/lumberjack/blob/554bab0378f22d2935cb8630e4c7e89f3fc8a908/clients/go/pkg/cloudlogging/processor.go#L128 doesn't override the `resource` field. Maybe we make both java and go consistent in setting the `resource` field. And based on our doc, this field can be auto generated by cloud logging. I don't see any value of overridden it by ourselves. Did I miss any context?

Reply:
In addition, this part causes null fields in json maps which triggers incomplete unmarshal due to limitation in protojson. With the limitation of https://github.com/golang/protobuf/issues/1313, we won't be able to verify the timestamp is set in our e2e test and our test is also fragile. Therefore, I try to make the test logs without any null value in json maps or json arrays.